introduction/ankur/about.md

```diff
@@ -0,0 +1 @@
+This is ankur.
```

Comment:
Why has this file been added, please remove this and create only one directory with the github username.